app/src/main/AndroidManifest.xml

```diff
@@ -53,6 +53,8 @@
             android:name=".task.StarActivity"
             android:screenOrientation="fullSensor" />
         <activity android:name=".task.LessonCompleteActivity"></activity>
+        <activity android:name=".authentication.CameraViewActivity">
```

Comment:
Is this activity for _authentication_ or _training_, or both? I see it's in the "authentication" package.